packages/iris-grid/src/IrisGridModel.js

```diff
@@ -199,6 +199,10 @@ class IrisGridModel extends GridModel {
     throw new Error('set totalsConfig not implemented');
   }
 
+  get layoutHints() {
```

Comment:
Can you add JsDocs for this? What kind of object it returns, etc.
Note it could return `undefined` (in the case the API doesn't support `getLayoutHints` yet)